### PR TITLE
adb: add Spanish translation

### PR DESCRIPTION
**Pull request by moisesnandres**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/adb.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+`adb kill-server`
+
+- Inicia una terminal remota en la instance del emulador/dispositivo de destino:
+
+`adb shell`
+
+- Instala una aplicación Android a un emulador/dispositivo:
+
+`adb install -r {{path/to/file.apk}}`
```
Comment (navarroaxel) [inline pages.es/common/adb.md:20]: Please translate the arguments for get this easier for non-English speakers:
```suggestion
`adb install -r {{ruta/al/archivo.apk}}`
```

File: pages.es/common/adb.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Copia un archivo/directorio desde el dispositivo de destino:
+
+`adb pull {{path/to/device_file_or_directory}} {{path/to/local_destination_directory}}`
```
Comment (navarroaxel) [inline pages.es/common/adb.md:24]:
```suggestion
`adb pull {{ruta/al/archivo_o_directorio_en_el_dispositivo}} {{ruta/al/directorio_de_destino_local}}`
```

File: pages.es/common/adb.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+- Copia un archivo/directorio al dispositivo de destino:
+
+`adb push {{path/to/local_file_or_directory}} {{path/to/device_destination_directory}}`
```
Comment (navarroaxel) [inline pages.es/common/adb.md:28]:
```suggestion
`adb push {{ruta/al/archivo_o_directorio_local}} {{ruta/al/directorio_de_destino_en_el_dispositivo}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Thanks @moisesnandres. I just made some comments.

Remember that for translations you can send more than 1 translated page in 1 PR, just keep 1 commit per file. This doesn't apply to the primary «English» pages.

**Comment by moisesnandres** [comment]
@navarroaxel ok, I will add the next translations on this PR

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

Looks ok to me after @navarroaxel's comments are addressed

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**